sqlmesh/core/state_sync/base.py

```diff
@@ -64,6 +64,10 @@ def _schema_version_validator(cls, v: t.Any) -> int:
     importlib.import_module(f"sqlmesh.migrations.{migration}")
     for migration in sorted(info.name for info in pkgutil.iter_modules(migrations.__path__))
 ]
+PRE_CHECKS = {
+    pre_check: importlib.import_module(f"sqlmesh.pre_checks.{pre_check}")
```

Comment:
Can pre-checks share the module with the migration function? I'm not sure I want to have them separated

Reply:
Discussed offline– will be moving forward with this ^ approach instead.